<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1093</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>309</width>
      <height>139</height>
     </rect>
    </property>
    <property name="title">
     <string>Dataset</string>
    </property>
    <widget class="QPushButton" name="loadDirectory">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Directory</string>
     </property>
    </widget>
    <widget class="QComboBox" name="directoryBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="patientBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="sujeitoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Sujeito:</string>
     </property>
    </widget>
    <widget class="QLabel" name="atividadeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Atividade:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>30</y>
      <width>431</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Dados</string>
    </property>
    <widget class="QLabel" name="labelTipoAtividade">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Atividade Atual:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelAtividadeDado">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1670</x>
      <y>50</y>
      <width>211</width>
      <height>108</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="startbutton">
       <property name="text">
        <string>START</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopbutton">
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SET">
       <property name="text">
        <string>SET</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>309</width>
      <height>139</height>
     </rect>
    </property>
    <property name="title">
     <string>LOGGER</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>40</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Dados</string>
    </property>
    <widget class="QLabel" name="labelPNG">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>101</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="labelPolitecnica">
    <property name="geometry">
     <rect>
      <x>1750</x>
      <y>20</y>
      <width>209</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>POLITÃ‰CNICA PUCRS</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>309</width>
      <height>139</height>
     </rect>
    </property>
    <property name="title">
     <string>Gateway config</string>
    </property>
    <widget class="QPushButton" name="connectbutton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>129</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="iplabel">
        <property name="text">
         <string>IP:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ipTextBox"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>61</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="portlabel">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="portTextBox"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>144</y>
      <width>831</width>
      <height>901</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_33">
       <property name="title">
        <string>acc_x</string>
       </property>
       <widget class="QCustomPlot" name="acc_x_widget" native="true">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>19</y>
          <width>811</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_34">
       <property name="title">
        <string>acc_y</string>
       </property>
       <widget class="QCustomPlot" name="acc_y_widget" native="true">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>19</y>
          <width>811</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_35">
       <property name="title">
        <string>acc_z</string>
       </property>
       <widget class="QCustomPlot" name="acc_z_widget" native="true">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>841</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_36">
       <property name="title">
        <string>gyr_x</string>
       </property>
       <widget class="QCustomPlot" name="gyr_x_widget" native="true">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>29</y>
          <width>841</width>
          <height>131</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_37">
       <property name="title">
        <string>gyr_y</string>
       </property>
       <widget class="QCustomPlot" name="gyr_y_widget" native="true">
        <property name="geometry">
         <rect>
          <x>2</x>
          <y>21</y>
          <width>821</width>
          <height>151</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>939</x>
      <y>139</y>
      <width>971</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>gyr_z</string>
    </property>
    <widget class="QCustomPlot" name="gyr_z_widget" native="true">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>29</y>
       <width>941</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>340</y>
      <width>961</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>azimuth</string>
    </property>
    <widget class="QCustomPlot" name="azimuth_widget" native="true">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>29</y>
       <width>941</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>520</y>
      <width>961</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>pitch</string>
    </property>
    <widget class="QCustomPlot" name="pitch_widget" native="true">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>29</y>
       <width>941</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>700</y>
      <width>961</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>roll</string>
    </property>
    <widget class="QCustomPlot" name="roll_widget" native="true">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>29</y>
       <width>941</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
